app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -82,6 +84,21 @@ function AcuantCaptureCanvas({
 
   return (
     <>
+      {!isReady && (
+        <img
+          src={getAssetPath('spinner.gif')}
+          srcSet={`${getAssetPath('spinner.gif')}, ${getAssetPath('spinner@2x.gif')} 2x`}
```

Comment:
1. Same feedback as other PR, WDYT about multiple lines?

```
srcSet={[
  getAssetPath('spinner.gif'),
  `${getAssetPath('spinner@2x.gif')} 2x`
].join(', ')}
```

Reply:
> 1. Same feedback as other PR, WDYT about multiple lines?

I'd started to implement this, but... since it results in shorter than 100 column length when inlined, Prettier will collapse it back down to a single line.

We could do something where we use multi-line template strings:

```jsx
<img
  src={getAssetPath('spinner.gif')}
  srcSet={`
    ${getAssetPath('spinner.gif')},
    ${getAssetPath('spinner@2x.gif')} 2x`}
  alt=""
  width="144"
  height="144"
  style={{
    position: 'absolute',
    left: '50%',
    top: '50%',
    transform: 'translate(-72px, -72px)',
  }}
/>
```

Or:

```jsx
<img
  src={getAssetPath('spinner.gif')}
  srcSet={`
    ${getAssetPath('spinner.gif')},
    ${getAssetPath('spinner@2x.gif')} 2x
  `}
  alt=""
  width="144"
  height="144"
  style={{
    position: 'absolute',
    left: '50%',
    top: '50%',
    transform: 'translate(-72px, -72px)',
  }}
/>
```

Reply:
Meh it's probably not worth it --- if we get to a more complex `srcset` in the future we can figure this out, maybe just put it back on the one line for now

Reply:
Actually I like the first option but again, not a huge deal

Reply:
Sorry, I didn't see the follow-up comments until after merge. I ended up keeping what was proposed in https://github.com/18F/identity-idp/pull/4113#discussion_r475834640 as the second example option (updated in aa1afb1).